index.bs

```diff
@@ -2410,45 +2408,91 @@ To <dfn>check if an [=attribution source=] exceeds the time-based destination li
-    [=attribution source/reporting origin=].
 1. Let |destinations| be the [=set=] of every [=attribution rate-limit record/attribution destination=] in |matchingSources|,
     [=set/union|unioned=] with |source|'s [=attribution source/attribution destinations=].
-1. Let |sameReportingDestinations| be the [=set=] of every [=attribution rate-limit record/attribution destination=] in |matchingSameReportingSources|,
+1. Return whether |destinations|'s [=set/size=] is greater than [=max destinations per rate-limit window=].
+
+To <dfn>check if an [=attribution source=] exceeds the time-based destination per reporting site limit</dfn> given an
+[=attribution source=] |source|, run the following steps:
+
+1. Let |matchingSources| be all [=attribution rate-limit records=] |record| in the [=attribution rate-limit cache=] where all of the following are true:
+
+     * |record|'s [=attribution rate-limit record/scope=] is "<code>[=rate-limit scope/source=]</code>"
+     * |record|'s [=attribution rate-limit record/source site=] and |source|'s [=attribution source/source site=] are equal
+     * |record|'s [=attribution rate-limit record/reporting origin=] and |source|'s [=attribution source/reporting origin=] are [=same site=]
+     * |record|'s [=attribution rate-limit record/expiry time=] is greater than |source|'s [=attribution source/source time=]
+     * The [=duration from=] |record|'s [=attribution rate-limit record/time=] and |source|'s [=attribution source/source time=]
+        is less than [=destinations-per-reporting-site rate-limit window=]
+1. Let |destinations| be the [=set/union=] of all [=attribution rate-limit record/attribution destination=] in |matchingSources|,
     [=set/union|unioned=] with |source|'s [=attribution source/attribution destinations=].
-1. Let |hitRateLimit| be whether |destinations|'s [=set/size=] is greater than [=max destinations per rate-limit window=][0].
-1. Let |hitSameReportingRateLimit| be whether |sameReportingDestinations|'s [=set/size=] is greater than [=max destinations per rate-limit window=][1].
-1. If (|hitRateLimit|, |hitSameReportingRateLimit|) is
-    <dl class="switch">
-    : (false, false)
-    :: Return "<code>[=destination rate-limit result/allowed=]</code>".
-    : (false, true)
-    :: Return "<code>[=destination rate-limit result/hit reporting limit=]</code>".
-    : (true, false)
-    :: Return "<code>[=destination rate-limit result/hit global limit=]</code>".
-    : (true, true)
-    :: Return "<code>[=destination rate-limit result/hit reporting limit=]</code>".
+1. Return whether |destinations|'s [=set/size=] is greater than [=max destinations per reporting site per rate-limit window=].
+
+An [=attribution rate-limit record=] |a| <dfn for="attribution rate-limit record">is less than</dfn>
+an [=attribution rate-limit record=] |b| if the following steps return true:
+
+1. If |a|'s [=attribution rate-limit record/scope=] is less than |b|'s [=attribution rate-limit record/scope=],
```

Comment:
The spec says
```
A rate-limit scope is one of the following:
"source"
"event-attribution"
"aggregatable-attribution"
```
Just to clarify -- is this comparing these? Why?

Reply:
In the algorithm, the comparison is done only for `source` scope, so it's actually irrelevant, but just to make `less than` more complete.

Reply:
If it's irrelevant, I'd argue not to include it.